pingcap/tidb
# telemetry: fix the bug about unnecessary error when run tidb only

### What problem does this PR solve?


Problem Summary:

### What is changed and how it works?


What's Changed:

fix the bug about unnecessary error when run tidb only

How it Works:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
run tidb only and observe


Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

- fix the bug about unnecessary error when run tidb only
